graylog2-server/src/main/java/org/graylog/events/notifications/types/EmailSender.java

```diff
@@ -104,6 +103,21 @@ private String buildBody(EmailEventNotificationConfig config, Map<String, Object
         return this.templateEngine.transform(template, model);
     }
 
+    @VisibleForTesting
+    private String buildHtmlBody(EmailEventNotificationConfig config, Map<String, Object> model) {
+        final String template;
+        if (isNullOrEmpty(config.htmlBodyTemplate())) {
```

Comment:
I think we should respect that user's wish and not fill in the default body if that is left empty.
(see next comment on how I would handle this)